sample-project/vite.config.js

```diff
@@ -11,6 +11,6 @@ export default defineConfig({
   plugins: [
     Inspect(),
     fable({ jsx: "automatic" }),
-    react({ include: /\.fs$/ }),
+    react({ include: /\.fs$/, jsxRuntime: "classic" }),
```

Comment:
Why this change?

Reply:
For Feliz in sample project, without this I saw it didn’t detect HMR properly but would make full page reloads with the new Feliz addition (doesn’t have Feliz compiler service but maybe is a transitive dep?)

Reply:
Then it makes no sense to keep `fable({ jsx: "automatic" })`, go with `fable()` instead.